README.md

```diff
@@ -1,2 +1,138 @@
+  - In "Designer" section, add a CloudWatch Events trigger
+	  - Set CloudWatch to trigger at your desired ingestion frequency. Once every 15 minutes is the highest frequency you should set for work zone status ingestion.
+	- In "Function code" section, select "Upload a .zip file" and upload the `wzdx_ingest.zip` file as your "Function Package."
+	- In "Environment variables" section, set the following:
+	  - `BUCKET`: the destination s3 bucket where the WZDx feed should be archived to.
+		  - default set as: usdot-its-workzone-public-data
+		- `FEED`: stringified json object containing information about the WZDx feed to ingest. At a minimum, this should include `state`, `feedName`, and `url`.
+	    - default set as: {"state": "arizona", "feedName": "mcdot", "url": "http://api.mcdot-its.com/WZDx/Activity/Get"}
+	- In "Basics settings" section, set adequate Memory and Timeout values. Memory of 1664 MB and Timeout value of 10 minutes should be plenty.
+4. Make sure to save all of your changes.
+
+### Deployment of S3 Explorer site
+
+1. Upload `index.html` to the root folder of your S3 bucket.
+2. In the AWS Console for your S3 bucket, go to "Permissions" > "CORS configuration" and copy and paste the following block of text and replace `{YOUR_WORKZONE_BUCKET_NAME}` with your bucket name.
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+<CORSRule>
+    <AllowedOrigin>*</AllowedOrigin>
+    <AllowedOrigin>http://{YOUR_WORKZONE_BUCKET_NAME}.s3.amazonaws.com</AllowedOrigin>
+    <AllowedOrigin>https://s3.amazonaws.com</AllowedOrigin>
+    <AllowedMethod>GET</AllowedMethod>
+    <AllowedMethod>HEAD</AllowedMethod>
+    <MaxAgeSeconds>3000</MaxAgeSeconds>
+    <ExposeHeader>ETag</ExposeHeader>
+    <ExposeHeader>x-amz-meta-custom-header</ExposeHeader>
+    <AllowedHeader>Authorization</AllowedHeader>
+    <AllowedHeader>*</AllowedHeader>
+</CORSRule>
+</CORSConfiguration>
+```
+
+3. Save. Also make sure that your bucket policy allows for List/Get actions on resource `arn:aws:s3:::{YOUR_WORKZONE_BUCKET_NAME}/*` and `arn:aws:s3:::{YOUR_WORKZONE_BUCKET_NAME}`.
+
+
+## Built With
+
+* [Python 2.7 or 3.x](https://www.python.org/download/releases/2.7/, https://www.python.org/download/releases/3.0) :
+* [requests](https://pypi.org/project/requests/) : package managing HTTP requests
+* [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html?id=docs_gateway) : AWS API
+
+## Contributing
+
+1. Fork it (https://github.com/usdot-its-jpo-data-portal/cv_pilot_ingest/fork)
+2. Create your feature branch (git checkout -b feature/fooBar)
+3. Commit your changes (git commit -am 'Add some fooBar')
+4. Push to the branch (git push origin feature/fooBar)
+5. Create a new Pull Request
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for general good practices on code of conduct, and the process for submitting pull requests.
+
+## Versioning
```

Comment:
I would say take out the versioning section unless we want to start using releases

Reply:
Removed this section as well.